system/user/addons/wb_category_select/ft.wb_category_select.php

```diff
@@ -338,11 +366,11 @@ public function grid_display_field($data)
 	private function _build_field($data, $cell = FALSE)
```

Comment:
Since you've expanded what `$cell` means, you should change the documentation of the function.